x-pack/plugins/cloud_security_posture/public/components/cloud_security_data_table/fields_selector.tsx

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useState, useMemo } from 'react';
+import {
+  EuiBasicTableColumn,
+  EuiButton,
+  EuiCheckbox,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiInMemoryTable,
+  EuiModal,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiSearchBarProps,
+  EuiText,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { type DataView } from '@kbn/data-views-plugin/common';
+
+interface Field {
+  id: string;
+  name: string;
+  displayName: string;
+}
+
+export interface FieldsSelectorCommonProps {
+  dataView: DataView;
+  columns: string[];
+  onAddColumn: (column: string) => void;
+  onRemoveColumn: (column: string) => void;
+}
+
+const ACTION_COLUMN_WIDTH = '24px';
```

Comment:
Maybe use preset value for this ?

Reply:
it was actually intentional not to use tokens in that case, as tokens can change and here we want a fixed value.